packages/interface/src/config.ts

```diff
@@ -68,6 +73,19 @@ export const getRPCURL = (): string | undefined => {
   }
 };
 
+/**
+ * Get the PIMLICO RPC URL based on the network we are connected to
+ * @returns the PIMLICO RPC URL
+ */
+export const getPimlicoRPCURL = (): string | undefined => {
+  switch (process.env.NEXT_PUBLIC_CHAIN_NAME) {
+    case "optimismSepolia":
+      return `https://api.pimlico.io/v2/optimism-sepolia/rpc?apikey=${process.env.NEXT_PUBLIC_PIMLICO_API_KEY}`;
```

Comment:
aren't endpoints using the chainId? i.e. `https://api.pimlico.io/v2/10/rpc?apiKey`?

Reply:
Didn't know it works with chainId too. Looks like it works with both https://docs.pimlico.io/permissionless/how-to/accounts/use-kernel-account#create-the-clients